doc-src/install.rst

```diff
@@ -1,25 +1,330 @@
 Installation
 ************
 
-This section describes how to build and install Charliecloud. For some
-distributions, this can be done using your package manager; otherwise, both
-normal users and admins can build and install it manually.
+This section describes what you need to install Charliecloud and how to do so.
+
+Note that installing and using Charliecloud requires no privilege, provided
```

Comment:
> * This statement is assuming autotools and related dependencies are installed?

Autotools are only needed for maintainers or people using the Git clone, but not, they don't require privileges.

> * Some of the image builders and their dependencies do (e.g., skopeo, unless you build with spack) require sudo. I think this should be mentioned here or earlier.

Let's mention that in the context of the builders. This will all be complemented by a prior section on the Charliecloud workflow and philosophy, which will cover the builders and why you might choose one or the other.